ark-sample/src/main.rs

```diff
@@ -363,6 +396,128 @@ async fn main() -> Result<()> {
                 }
             }
         }
+        Commands::BatchSend {
+            addresses_and_amounts,
+        } => {
```

Comment:
Can the send-to-a-single address and this new branch be merged? Surely sending to a single address can be handled here.